src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The class should not start with an empty line. According to the checklist, don't begin class or method implementation with an empty line.